tests/unit/tuf_repository_service_worker/test_repository.py

```diff
@@ -330,11 +330,19 @@ def test__update_snapshot(self, test_repo):
         test_repo._storage_backend.get = pretend.call_recorder(
             lambda *a: mocked_snapshot
         )
-        test_repo._bump_and_persist = pretend.call_recorder(lambda *a: None)
+
+        def fake__bump_and_persist(md, role, **kw):
+            if role == Snapshot.type:
+                md.signed.version += 1
+
+        test_repo._bump_and_persist = pretend.call_recorder(
+            fake__bump_and_persist
+        )
 
         result = test_repo._update_snapshot()
 
-        assert result is snapshot_version
+        assert result == 4
+        assert mocked_snapshot.signed.version == 4
```

Comment:
Seems more logical to do that instead of hardcoding values and it's what exactly we expect to happen semantically:
```suggestion
        assert result == snapshot_version + 1
        assert mocked_snapshot.signed.version == snapshot_version + 1
```